### PR TITLE
Adding strato-log search in default paths and default bundles - review

### PR DESCRIPTION
**Pull request by sergey-stratoscale**
@Stratoscale/systegration 
default paths: checking if provided file exists, if not, generate list of files to scan according to every file in default dirs that match the file.
default bundles: if provided default regex, create list of files that match that bundle. in that case, the only bundle i've added will bring all vm's consoles if they exist

config file : https://github.com/Stratoscale/loggingtools/pull/9

### CODE REVIEW

File: py/strato/common/log/log2text.py [hunk 1]
```diff
@@ -233,6 +235,53 @@ def updateConfFile(field, value):
...
+        if not self.defaultPaths:
+            self._getDefaultPaths()
+        matchingFiles = [defaultFilePath for defaultFilePath in self.defaultPaths
+                         if filePath.lower() in defaultFilePath.lower() and os.path.isfile(defaultFilePath)]
+        if not matchingFiles:
+            self._raiseNoMatchError(filePath)
+        else:
+            return matchingFiles
+
+    def _raiseNoMatchError(self, filePath):
```
Comment (rom-stratoscale) [inline py/strato/common/log/log2text.py]: why do u need a function for this?

Comment (sergey-stratoscale) [inline py/strato/common/log/log2text.py]: because i throw the same error in 2 places

Comment (rom-stratoscale) [inline py/strato/common/log/log2text.py]: So you should raise same exception. Why to use a function?

File: py/strato/common/log/log2text.py [hunk 2]
```diff
@@ -233,6 +235,53 @@ def updateConfFile(field, value):
...
+        raise Exception("No matching files were found for %s" % filePath)
+
+    def _getDefaultPaths(self):
+        for filePath in self.confFile['defaultPaths']:
+            self.defaultPaths.extend(glob.glob(filePath))
```
Comment (romang-stratoscale) [inline py/strato/common/log/log2text.py]: globglob

File: py/strato/common/log/log2text.py [hunk 3]
```diff
@@ -233,6 +235,53 @@ def updateConfFile(field, value):
...
+                         if filePath.lower() in defaultFilePath.lower() and os.path.isfile(defaultFilePath)]
+        if not matchingFiles:
+            self._raiseNoMatchError(filePath)
+        else:
+            return matchingFiles
+
+    def _raiseNoMatchError(self, filePath):
+        raise Exception("No matching files were found for %s" % filePath)
+
+    def _getDefaultPaths(self):
```
Comment (rom-stratoscale) [inline py/strato/common/log/log2text.py]: I think I lost the flow here. You have something in init, than you have default from another place... why do you need al this mismatch?

Comment (sergey-stratoscale) [inline py/strato/common/log/log2text.py]: default path in init is a property that is set to an empty list. this function sets that property's value according to what is written in config file

**Pull request merged**